homeassistant/components/vizio/media_player.py

```diff
@@ -88,31 +100,32 @@ def __init__(self, host, token, name, volume_step, device_type):
         self._available_inputs = None
         self._device_type = device_type
         self._supported_commands = SUPPORTED_COMMANDS[device_type]
-        self._device = Vizio(DEVICE_ID, host, DEFAULT_NAME, token, device_type)
+        self._device = VizioAsync(DEVICE_ID, host, DEFAULT_NAME, token, device_type)
```

Comment:
Please update the library to have it accept and use an optional aiohttp session. We want all libraries that use aiohttp to use the home assistant aiohttp client session. It improves performance.

Reply:
Get the home assistant session with this helper:
https://github.com/home-assistant/home-assistant/blob/6b7be35f4a10733fad85fc16a65bd39e02059c2d/homeassistant/helpers/aiohttp_client.py#L30